From 59d505316668f5c89918a1060e758cb302cb35f2 Mon Sep 17 00:00:00 2001
From: RickyWu-wiwynn <ricky_cx_wu@wiwynn.com>
Date: Fri, 1 Nov 2024 13:50:21 +0800
Subject: [PATCH 1001/1012] arm: dts: aspeed: yosemite4: add mac config property

Add mac config property to fix the issue that Mellanox CX7 cannot get IP after NIC re-plug.
- Haven't contributed because it is depend on following patch:
https://github.com/facebookexternal/openbmc.wiwynn/blob/542f21879cb950b22d9792269aa640a354985848/common/recipes-kernel/linux/6.6/0183-NCSI-Add-propety-no-channel-monitor-and-start-redo-p.patch
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
index 7937ca90cb05..d50be1fe08a3 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
@@ -135,6 +135,8 @@ &mac2 {
 	pinctrl-0 = <&pinctrl_rmii3_default>;
 	use-ncsi;
 	mellanox,multi-host;
+	ncsi-ctrl,start-redo-probe;
+	ncsi-ctrl,no-channel-monitor;
 };
 
 &mac3 {
@@ -143,6 +145,8 @@ &mac3 {
 	pinctrl-0 = <&pinctrl_rmii4_default>;
 	use-ncsi;
 	mellanox,multi-host;
+	ncsi-ctrl,start-redo-probe;
+	ncsi-ctrl,no-channel-monitor;
 };
 
 &fmc {
-- 
2.25.1

